classes/Validate.php

```diff
@@ -279,7 +279,7 @@ public static function isImageTypeName($type)
      */
     public static function isPrice($price)
     {
-        return preg_match('/^[0-9]{1,10}(\.[0-9]{1,9})?$/', $price);
+        return preg_match('/^[0-9]{1,10}(\.[0-9]*})?$/', $price);
```

Comment:
Since it was `{1,9}`, we need One or more.
```suggestion
        return preg_match('/^[0-9]{1,10}(\.[0-9]+})?$/', $price);
```